api_tests/chronos/views/test_chronos_submission_list.py

```diff
@@ -73,7 +74,10 @@ def other_submission(self, other_preprint, journal_one, submitter):
 
     @pytest.fixture()
     def url(self, preprint):
-        return f'/_/chronos/{preprint._id}/submissions/'
+        preprint_id = preprint._id
+        if VersionedGuidMixin.GUID_VERSION_DELIMITER in preprint._id:
+            preprint_id = preprint._id.split(VersionedGuidMixin.GUID_VERSION_DELIMITER)[0]
+        return f'/_/chronos/{preprint_id}/submissions/'
```

Comment:
hmm, instead of modifying the test to use base guid, we should update the api code to support versioned guid.